distributed/deploy/cluster.py

```diff
@@ -70,8 +91,24 @@ async def _start(self):
         self._watch_worker_status_task = asyncio.ensure_future(
             self._watch_worker_status(comm)
         )
+
+        with contextlib.suppress(KeyError):  # The scheduler might not have any info
+            self.cluster_info.update(
+                (await self.scheduler_comm.get_metadata(keys=["cluster-manager-info"]))
+            )
+        self.loop = IOLoop.current()
+        self.periodic_callbacks["sync-cluster-info"] = pc = PeriodicCallback(
+            self._sync_cluster_info, self._sync_interval * 1000
+        )
+        self.loop.add_callback(pc.start)
```

Comment:
I could be missing something, but it would be more consistent with what we do elsewhere if we just call `pc.start()` for all the callbacks in `self.periodic_callbacks`. This will also be more robust to adding future callbacks to `self.periodic_callbacks` and _should_ handle attaching to the correct event loop. 

```diff
diff --git a/distributed/deploy/cluster.py b/distributed/deploy/cluster.py
index 7e46ecc9..c9862c00 100644
--- a/distributed/deploy/cluster.py
+++ b/distributed/deploy/cluster.py
@@ -8,7 +8,7 @@ import uuid
 from contextlib import suppress
 from inspect import isawaitable

-from tornado.ioloop import IOLoop, PeriodicCallback
+from tornado.ioloop import PeriodicCallback

 import dask.config
 from dask.utils import _deprecated, format_bytes, parse_timedelta
@@ -96,11 +96,11 @@ class Cluster:
             self.cluster_info.update(
                 (await self.scheduler_comm.get_metadata(keys=["cluster-manager-info"]))
             )
-        self.loop = IOLoop.current()
-        self.periodic_callbacks["sync-cluster-info"] = pc = PeriodicCallback(
+        self.periodic_callbacks["sync-cluster-info"] = PeriodicCallback(
             self._sync_cluster_info, self._sync_interval * 1000
         )
-        self.loop.add_callback(pc.start)
+        for pc in self.periodic_callbacks.values():
+            pc.start()
         self.status = Status.running

     async def _sync_cluster_info(self):
```

Reply:
Thanks @jrbourbeau!